### ISSUE TITLE
Docs: How far we want to take it?

### ISSUE THREAD

**Issue by svx** [plone/plone.docker#59]
This is now more my personal opinion, but I would like feedback, before I start writing stuff.

Here it comes:

I do not want to dive too deep into various topics, because often it is depending on your infra, personal taste and so on.

For example topics like, restarting, booting and so on ...

IMHO we should add to the beginning of the docs some short lines like:

> If you going to use docker in production, we assume that you know what you are doing, bla bla

I am +1 for giving some examples for how you run a test container for playing, but for more serious stuff people should know what they are doing.

If you use rancher, swarm, dc/os, kubernetes, this is on you and not topic of the docs.

**Comment by demarant** [comment]
I agree. We should not describe in details on how to deploy with specific clustering tools like rancher, swarm, dc/os, kubernetes. We should just explain the basic docker commands. For advanced clustering, if we know that some good examples exists, I would be in favor of "just" add some good stable references.
